src/inmanta_module_factory/builder.py

```diff
@@ -227,19 +228,12 @@ def test_basics(project: Project) -> None:
 
     def generate_module(
         self, build_location: Path, force: bool = False, copyright_header_template: Optional[str] = None
-    ) -> None:
-        module_path = build_location / self._module.name
-        if module_path.exists():
-            if not force:
-                raise RuntimeError(f"Generating this module would have overwritten the following path: {str(module_path)}")
```

Comment:
So we don't need this check anymore? why?

Reply:
If so I think the "force" argument can be removed from the function def

Reply:
We don't need it because cookiecutter will do the check for us, see the `overwrite_if_exists` option we have added below.  (So we still use this `force` parameter)